cmd/world/forge/deployment.go

```
@@ -128,3 +133,130 @@
+	if data["project_id"] != projectID {
+		return eris.Errorf("Deployment status does not match project id %s", projectID)
+	}
+	if data["type"] != "deploy" {
+		return eris.Errorf("Deployment status does not match type %s", data["type"])
+	}
+	executorID := data["executor_id"].(string)
+	dt, dte := time.Parse(time.RFC3339, data["execution_time"].(string))
+	if dte != nil {
+		return eris.Wrapf(dte, "Failed to parse execution time %s", dt)
+	}
+	buildNumber := int(data["build_number"].(float64))
```

golangci-lint (1.62.2)

Error return value is not checked

(errcheck)